recipes/epispot/meta.yaml

```
@@ -0,0 +1,41 @@
+{% set name = "epispot" %}
+{% set version = "2.0.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/epispot/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: 317c3d9ccdd012e173942e8953bebff2171b1d4ab100ff9ccc54cec648e7c197
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python >=3.6
+    - pip
+    - numpy
+    - matplotlib
+  run:
+    - python >=3.6
+    - numpy
+    - matplotlib

```

https://conda-forge.org/docs/maintainer/knowledge_base.html#matplotlib

```suggestion
    - matplotlib-base
```